cgm/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf -fi
+./configure --prefix=$PREFIX --enable-optimize --disable-debug --enable-shared
```

Comment:
Ahh I see.  So should I just get rid of the cmg recipe?  It seems like it might be useful here eventually if OpenCascade CE ever works.